packages/system/metallb/Makefile

```
@@ -9,3 +10,25 @@ update:
 	helm repo update metallb
 	helm pull metallb/metallb --untar --untardir charts
 	rm -rf charts/metallb/charts/frr-k8s
+
+image-controller image-speaker:
+	$(eval TARGET := $(subst image-,,$@))
+	$(eval VERSION := $(shell yq '.appVersion' charts/metallb/Chart.yaml))
+	docker buildx build images/metallb \
+		--provenance false \
+		--target $(TARGET) \
+		--build-arg VERSION=$(VERSION) \
+		--tag $(REGISTRY)/metallb/$(TARGET):$(VERSION) \
+		--cache-from type=registry,ref=$(REGISTRY)/metallb/$(TARGET):latest \
```

```suggestion
		--tag $(REGISTRY)/metallb-$(TARGET):$(VERSION) \
		--cache-from type=registry,ref=$(REGISTRY)/metallb-$(TARGET):latest \
```

It might be better to keep one-level for image names because some registries does not support multi-level notation